src/opening-period/form/OpeningPeriodForm.scss

```diff
@@ -35,6 +35,7 @@
 
 .form-actions-row-condensed {
   padding: 0 $spacing-l;
+  margin-bottom: $spacing-layout-2-xl;
```

Comment:
Could this be a variable based on the height of the floating action row? I assume with sticky position this is not even needed

Reply:
- [ ] Apply final action row container height